src/vega.schema.ts

```diff
@@ -90,13 +68,13 @@ export interface VgValueRef {
 // TODO: add vg prefix
 export interface DataRefUnionDomain {
   fields: (any[] | VgDataRef | SignalRef)[];
-  sort?: VgUnionSortField;
+  sort?: UnionSortField;
 }
 
 export interface VgFieldRefUnionDomain {
   data: string;
-  fields: VgFieldRef[];
-  sort?: VgUnionSortField;
+  fields: FieldValue[];
```

Comment:
The name FieldValue is pretty confusing.

Reply:
I think it should be `Field` in Vega typings, and we probably should call it `VgField` here.